src/Components/CollectorProfile/CollectorProfileArtistsAddDialog.tsx

```diff
@@ -11,11 +12,23 @@ interface CollectorProfileArtistsAddDialogProps {
 export const CollectorProfileArtistsAddDialog: FC<
   React.PropsWithChildren<CollectorProfileArtistsAddDialogProps>
 > = ({ onClose, onSuccess, title }) => {
+  const { submitUpdateMyUserProfile } = useUpdateMyUserProfile()
+
+  const handleClose = async () => {
+    try {
+      await submitUpdateMyUserProfile({ promptedForUpdate: true })
+    } catch (err) {
+      console.error(err)
+    }
+
+    onClose()
```

Comment:
I'm not sure if you need to `await`? I'd imagine this causes a delay when closed (try throttling your connection)?